server/protocols/webadmin/webadmin-mailrepository/pom.xml

```diff
@@ -42,6 +42,16 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-data-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-data-memory</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-data-library</artifactId>
```

Comment:
Alphabetic ordering & you should already have dependency data-lib with data-memory